### PR TITLE
feat(rad-964): new entity structure

### PR DESCRIPTION
**Pull request by michal-aylien**

### CODE REVIEW

File: aylien/v1/news/api-docs.yaml [hunk 1]
```diff
@@ -4766,20 +4786,104 @@ components:
...
           type: array
         title:
           description: An array of extracted entities from the story title
           items:
-            $ref: "#/components/schemas/Entity"
+            $ref: "#/components/schemas/DeprecatedEntity"
           type: array
       type: object
-    Entity:
+    DeprecatedEntity:
```
Comment (FrontSide) [inline aylien/v1/news/api-docs.yaml:4886]: Why do we need to support DeprecatedEntity here?

Comment (michal-aylien) [reply aylien/v1/news/api-docs.yaml:4886]: we need to - it will be returned for the users that do not have new entity format enabled

Comment (FrontSide) [reply aylien/v1/news/api-docs.yaml:4886]: How does it currently work with entity v2 vs entity v3 customers?

Comment (michal-aylien) [reply aylien/v1/news/api-docs.yaml:4886]: currently there exists one type to represent v2 and v3 entities
it is just filled differently with data

Comment (FrontSide) [reply aylien/v1/news/api-docs.yaml:4886]: ok makes sense

**Review by FrontSide — APPROVED** [review]

**Pull request merged**